RecoTauTag/RecoTau/test/rerunMVAIsolationOnMiniAOD_Phase2.py

```diff
@@ -24,9 +24,11 @@
 )
 
 from RecoTauTag.RecoTau.TauDiscriminatorTools import noPrediscriminants
-### Load PoolDBESSource with mapping to payloads
-# Note: replace it by an appropriate GlobalTag when MVAIso phase-2 payloads are available via it
-process.load('RecoTauTag.Configuration.loadRecoTauTagMVAsFromPrepDB_cfi')
+### Load payloads via GlobalTag
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, '112X_mcRun4_realistic_Candidate_2020_09_17_13_57_14', '')
```

Comment:
this GlobalTag candidate still has `PGeometricDetExtraRcd` which has been eliminated by cmssw in PR https://github.com/cms-sw/cmssw/pull/31729.
This leads to:
```
----- Begin Fatal Exception 20-Oct-2020 16:29:52 CEST-----------------------
An exception of category 'PluginLibraryLoadError' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing ESSource: class=PoolDBESSource label='GlobalTag'
   [2] While attempting to load plugin PGeometricDetExtraRcd@NewProxy
Exception Message:
unable to load /cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc820/cms/cmssw-patch/CMSSW_11_2_X_2020-10-19-2300/lib/slc7_amd64_gcc820/poisoned/plugin-poisoned-CondCoreGeometryPluginsPlugins.so because /cvmfs/cms-ib.cern.ch/week1/slc7_amd64_gcc820/cms/cmssw-patch/CMSSW_11_2_X_2020-10-19-2300/lib/slc7_amd64_gcc820/poisoned/plugin-poisoned-CondCoreGeometryPluginsPlugins.so: cannot open shared object file: No such file or directory
----- End Fatal Exception -------------------------------------------------
Failure using rerunMVAIsolationOnMiniAOD_Phase2.py: status 65
status = 16640

---> test runtestRecoTauTagRecoTau had ERRORS
```

please use `auto:phase2_realistic` instead.

Reply:
Right, this helps. Thanks a lot!